### PR TITLE
FAT-377044 Verify interface language (Firebird)

### PR DESCRIPTION
**Pull request by dnovikau**
[C377044](https://foliotest.testrail.io/index.php?/cases/view/377044)
***
[Allure report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4406/allure/)
***

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = defineConfig({
   defaultCommandTimeout: 101000,
   pageLoadTimeout: 120000,
   env: {
-    OKAPI_HOST: 'https://folio-testing-cypress-okapi.ci.folio.org',
+    OKAPI_HOST: 'https://folio-snapshot-okapi.dev.folio.org',
```
Comment (ALegchilov) [inline cypress.config.js:22]: Did you change it on purpose?

Comment (dnovikau) [reply cypress.config.js]: Fixed

**Review by nayimovag — APPROVED** [review]

File: cypress/e2e/settings/tenant/change-interface-language.cy.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      Localication.changeNumberingSystem(NUMBERS.LATN);
+      Localication.clickSaveButton();
+
+      // Click on the "Locale (for language display, date format etc.)" dropdown and select "English (Sweden) / English (Sweden)" language => Click "Save" button
+      Localication.changeLocalLanguage(LANGUAGES.SWEDEN_ENGLISH);
+      Localication.clickSaveButton();
+
+      // Click on the "Numbering system" dropdown and select "arab" option => Click "Save" button
+      Localication.changeNumberingSystem(NUMBERS.ARAB);
+      Localication.clickSaveButton();
```
[the remark this answers is not in the document]
Comment (dnovikau) [reply cypress/e2e/settings/tenant/change-interface-language.cy.js:54]: Fixed

File: cypress/e2e/settings/tenant/change-interface-language.cy.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+      // Click on the "Locale (for language display, date format etc.)" dropdown and select "American English/American English" language => Click "Save" button
+      Localication.changeLocalLanguage(LANGUAGES.AMERICAN_ENGLISH);
+
+      // Click on the "Numbering system" dropdown and select "---" (no) option => Click "Save" button
+      Localication.changeNumberingSystem(NUMBERS.NONE);
+      Localication.clickSaveButton();
+    },
+  );
+});
```
[the remark this answers is not in the document]
Comment (dnovikau) [reply cypress/e2e/settings/tenant/change-interface-language.cy.js:64]: Fixed

File: cypress/e2e/settings/tenant/change-interface-language.cy.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      // Select "Language and localization" option
+      TenantPane.selectTenant(TENANTS.LANGUAGE_AND_LOCALIZATION);
+      Localication.checkPaneContent();
+
+      // Click on the "Locale (for language display, date format etc.)" language
+      Localication.changeLocalLanguage(LANGUAGES.BRITISH_ENGLISH);
+
+      // Click on the "Numbering system" dropdown and select "latn (0 1 2 3 4 5 6 7 8 9)" option => Click "Save" button
+      Localication.changeNumberingSystem(NUMBERS.LATN);
+      Localication.clickSaveButton();
```
[the remark this answers is not in the document]
Comment (dnovikau) [reply cypress/e2e/settings/tenant/change-interface-language.cy.js:46]: Fixed

**Review by manowarqqq — APPROVED** [review]

**Pull request merged**